libvcs4j-tools/src/test/java/de/unibremen/informatik/st/libvcs4j/mapping/MappingTest.java

```diff
@@ -0,0 +1,521 @@
+package de.unibremen.informatik.st.libvcs4j.mapping;
+
+import de.unibremen.informatik.st.libvcs4j.*;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+
+public class MappingTest {
+
+    private Mapping<String> mapping;
+
+    private RevisionRange revisionRange;
+
+    @Before
+    public void setUp() {
+        mapping = new Mapping<>();
+        revisionRange = mock(RevisionRange.class);
+        Revision revision = mock(Revision.class);
+        when(revision.getId()).thenReturn("2");
+        when(revisionRange.getRevision()).thenReturn(revision);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
You should prefer the `ExpectedException Rule` rather than the `expected` parameter of `@Test`. See https://github.com/junit-team/junit4/wiki/exception-testing